### DISCUSSION TITLE
刚答完题选择题得分65分，是不是意味着我这周又不及格了？

### DISCUSSION THREAD

**Discussion by wangchuanyes** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#37] [General]
到底问题出在哪啊，看了这么多遍题，小心翼翼的答，怎么我就觉得我应该是满分呢？

**Comment by coldshineb** [comment]
因为答案没说是全对

**Comment by m1w22** [comment]
前几天84分，满分选择题，填空题人工审核后是89分


1.10的题库

4981298是不正确的

4981321是正确的

**Comment by MlgmXyysd** [comment]
应该是有错的，不显示得分对不了答案，你们有5级号的可以试错讨论讨论，已经通过审核的也可以去答题贡献一下

**Comment by Cyclonekid** [comment]
改完分    84的路过`   这是标准黑号了的节奏``
